cradle-cassandra/scripts/download-statistics-from-cassandra.sh

```diff
@@ -315,6 +322,7 @@ parse_args() {
   echo "  Arg: ${ARG_EVENT_ENTITY_STATISTICS_CSV} = ${FILE_EVENT_ENTITY_STATISTICS_CSV}"
   echo "  Arg: ${ARG_GROUPED_MESSAGES_CSV} = ${FILE_GROUPED_MESSAGES_CSV}"
   echo "  Arg: ${ARG_TEST_EVENTS_CSV} = ${FILE_TEST_EVENTS_CSV}"
+  echo "  Arg: ${ARG_EXCLUDE_REC_DATE} = ${EXCLUDE_REC_DATE:-y}"
```

Comment:
Please set `y` for EXCLUDE_REC_DATE variable when ARG_EXCLUDE_REC_DATE argument is handled instead of unset value for EXCLUDE_REC_DATE and use default value here

Reply:
If I do that the logic responsible for including column in the query will not work